docs/content/documentation/deployment/vercel.md

```diff
@@ -23,3 +23,85 @@ You can learn more about how to setup a custom domain and how to get the most ou
 [via their documentation.](https://vercel.com/docs) 
 
 After you click the blue "Deploy" button, it's off to the races!
+
+To use a specific version of Zola, set [`ZOLA_VERSION`](https://vercel.com/docs/deployments/environments#specifying-framework-versions) environment variable in project settings to a valid
+release tag, for example `0.17.2`.
+
+## Troubleshooting
+
+### `GLIBC_X.XX` not found
+
+This is because Vercel's build images comes with an older glibc version whereas Zola 
+depends on a newer glibc. However, Vercel provides a newer build image which can be used in
+deployments by setting Node.js version to "20.x", allowing Zola to work properly.
+
+## Additional options
+
+### Enable trailing slashes
+
+Visiting a page without trailing slash may break relative paths, so you might want to configure
+Vercel to always redirect paths with a trailing slash. By default, redirecting to a trailing
+slash is not enabled on Vercel.
+
+For example if you have an `about.md` file, and when visiting the path without a trailing
+slash, like `/about`, Vercel will redirect with trailing slash, resulting in `/about/`.
+Paths with a file extension will not redirect to a trailing slash, for example if you
+have a static file named `favicon.ico`, it will stay as-is.
+
+To enable that, create a file in the root of your git repository named `vercel.json`
+(if it doesn't exists already), and set this option:
+
+```json
+{
+    "trailingSlash": true
+}
+```
+
+### Prefer clean URLs
+
+When enabled, all HTML files will be served without their file extension. For example
+if you have an `about.md` file, Zola will generate a `about.html` file, but Vercel
```

Comment:
> Zola will generate a `about.html`

No, Zola will generate `about/index.html`

Reply:
Yeah, it was an oversight on my part, I forgot how it works for a moment 😅 Fixed.